testtools/iothub_test/src/iothub_account.c

```diff
@@ -55,6 +55,18 @@ const int TEST_SLEEP_THROTTLE_MSEC = 5 * 1000;
 const int TEST_SLEEP_BETWEEN_CREATION_FAILURES_MSEC = 30 * 1000;
 const int TEST_SLEEP_BETWEEN_METHOD_INVOKE_FAILURES_MSEC = 30 * 1000;
 
+#define NSLOOKUP_MAX_COMMAND_SIZE 128
+
+#ifndef popen
```

Comment:
Is a `#define` redirect necessary on Linux at all?

Reply:
Not on Linux. On Windows. On Linux it's popen, on Windows it's _popen...